### DISCUSSION TITLE
Is there a function of  pulsation?

### DISCUSSION THREAD

**Discussion by dingxu6207** [phoebe-project/phoebe2#559] [Questions]
Is there a function of pulsation. I run it with error?
b.add_feature('pulsation', 'primary', radamp=0.1, freq=5.0, feature='puls01')

**Comment by kecnry** [comment]
Pulsations are currently under development and not included in the released version of the code.  If you would like to play around with them or test, you can find them in the `pulsations` branch (but note that the implementation is not fully tested or released and should not be used for science at this point).
